src/java/main/org/apache/zookeeper/cli/SetAclCommand.java

```diff
@@ -35,10 +37,11 @@
     {
         options.addOption("s", false, "stats");
         options.addOption("v", true, "version");
+        options.addOption("R", false, "recursive");
     }
 
     public SetAclCommand() {
-        super("setAcl", "[-s] [-v version] path acl");
+        super("setAcl", "[-s] [-v version] [-R] path acl");
     }
```

Comment:
[-R recursive] is better?

Reply:
Since we don't have it for [-s] either, I think it's more consistent to keep [-R].

Reply:
- Since there is no documentation about setAcl,so the more detailed, the better.
- "[-s stat] [-v version] [-R recursive] path acl"

Reply:
And thanks for the code pointer.

Reply:
oops!I was wrong.